vms/evm/destination_client.go

```diff
@@ -9,27 +9,29 @@ import (
 	"context"
 	"math/big"
 	"sync"
+	"time"
 
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/utils/logging"
 	avalancheWarp "github.com/ava-labs/avalanchego/vms/platformvm/warp"
-	"github.com/ava-labs/coreth/rpc"
 	"github.com/ava-labs/icm-services/relayer/config"
 	"github.com/ava-labs/icm-services/utils"
 	"github.com/ava-labs/icm-services/vms/evm/signer"
 	"github.com/ava-labs/subnet-evm/core/types"
 	"github.com/ava-labs/subnet-evm/ethclient"
 	"github.com/ava-labs/subnet-evm/precompile/contracts/warp"
 	predicateutils "github.com/ava-labs/subnet-evm/predicate"
+	"github.com/ava-labs/subnet-evm/rpc"
 	"github.com/ethereum/go-ethereum/common"
 	"go.uber.org/zap"
 )
 
 const (
 	// Set the max fee to twice the estimated base fee.
 	// TODO: Revisit this constant factor when we add profit determination, or make it configurable
-	BaseFeeFactor        = 2
-	MaxPriorityFeePerGas = 2500000000 // 2.5 gwei
+	BaseFeeFactor                 = 2
+	MaxPriorityFeePerGas          = 2500000000 // 2.5 gwei
+	DefaultBlockAcceptanceTimeout = 30 * time.Second
```

Comment:
Should we move setting the default to `DestinationBlockchain.Validate()` like we have now for the block gas limit and max priority fee per gas? That will also simplify the initialization logic below.